README.md

```diff
@@ -21,3 +21,37 @@ be available for use.
 # Running the image Locally
 After building the image, you can run it locally using the following command: <br>
 `podman run -p 8080:8080 MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080`
+
+## How to add the image to opendatahub/JupyterHub
+
+* Once you have pushed your image to an image registry that is accessible by your JupyterHub instance,
+you can use the `ImageStream` template in this repository at `openshift/oc-imagestream.yaml`.
+
+
+```yaml
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true" # <-- setting it true makes it visible to ODH/JupyterHub
+  name: s2i-custom-notebook # <-- Update the ImageStream name here
```

Comment:
Since there is a note about the naming of ImageStram in ODH 0.5 and older, should this suggested name omit hte `s2i-` prefix?

Reply:
that's just to keep it consistent with the repo name and the image name

Reply:
That's fine, whatever feels more natural. We will add more labels in the future to cover things like name, description, capabilities..